packages/zosjobs/src/GetJobs.ts

```diff
@@ -360,4 +381,19 @@ export class GetJobs {
         Logger.getAppLogger().info("GetJobs.getSpoolContentCommon() parameters: " + parameters);
         return ZosmfRestClient.getExpectString(session, JobsConstants.RESOURCE + parameters, [Headers.TEXT_PLAIN_UTF8]);
     }
+
+    private static filterResultsByStatuses(jobs: IJob[], params: IGetJobsParms): IJob[] {
+        if (params?.status && params.status.toLowerCase() != 'active') {
+            const statusJobs = jobs.filter(job => {
+                const resultStatusLower = job.status.toLowerCase();
+                const paramStatusLower = params.status.toLowerCase();
+                if (resultStatusLower === paramStatusLower) {
+                    return job;
```

Comment:
doesn't the predicate (function) of the `.filter` expects a `boolean`?

Reply:
Yes. it worked because job is Truthy. That function has been updated.